addons/arsenal/functions/fnc_refresh.sqf

```diff
@@ -15,7 +16,9 @@
  *
  * Public: Yes
 */
-params [["_updateItems", true, [true]]];
+params [["_updateItems", true, [true]], ["_updateVirtualItems", false, [false]]];
+
+TRACE_2("",_updateItems,_updateVirtualItems);
 
 if (_updateItems) then {
```

Comment:
As we are going to be updating the display if `_updateVirtualItems` is true, no need to do something expensive twice.
```suggestion
if (_updateItems && !_updateVirtualItems) then {
```